app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
+require "application_responder"
```

Comment:
You don´t need this require since there is a `responder` already loaded in the path from `app/responders`.